src/main/resources/db/migration/V1_0_19__create-table-kontantstotte.sql

```diff
@@ -0,0 +1,26 @@
+-- Table: kontantstotte
+
+-- DROP TABLE kontantstotte
+
+CREATE TABLE IF NOT EXISTS kontantstotte
+(
+    kontantstotte_id integer NOT NULL GENERATED BY DEFAULT AS IDENTITY ( INCREMENT 1 START 1 MINVALUE 1 MAXVALUE 2147483647 CACHE 1 ),
+    grunnlagspakke_id integer NOT NULL,
+    part_person_id varchar(50) not null,
+    barn_person_id varchar(50) NOT NULL,
+    periode_fra date NOT NULL,
+    periode_til date NOT NULL,
```

Comment:
Denne må være nullable

Reply:
Enig. Jeg ser at dette også må endres i skattegrunnlag og ainntekt tabellene også.

Reply:
Fpr Ainntekt og Skattegrunnlag så vil aldri periodeTil være null. Vi setter denne alltid til periodeFra + 1 måned, så ikke behov for å endre disse.